Versions/Common/BeStride_Variables.lua

```diff
@@ -61,6 +61,9 @@ BeStride_Variables = {
 			{
 				{name="classes.shaman.ghostwolf",element="CheckBox",label=BeStride_Locale.Settings.Classes.Shaman.GhostWolf,class="shaman"},
 			},
+			{
+				{name="classes.evoker.hover",element="CheckBox",label=BeStride_Locale.Settings.Classes.Evoker.Hover,class="evoker"},
+			},
```

Comment:
There is going to be a change to how options are done in the next version.  I am going to hold off on merge until after that is completed.

Reply:
No problem on holding off. Feel free to reject this PR and add Hover yourself if you want, or I can update/remake it later after the GUI changes. I mostly did this to ensure I'd have hover for my Evoker starting today.

Also, I've got some code changes on my fork with support for Dragonriding. That is something I'd love to have in the addon on the 28th. Was waiting to PR those until after today's patch since that changed some API calls (again) in retail. Also, it may break WotLK/Classic support? Not sure, will have to see...will take a look at that this weekend maybe.

For the translations, a little bit of Google Translate, but for the spell/ability names and Evoker name, I used Wowhead and flipped between the different language versions of the site. Spell ids remain the same, so getting the name for the ability in-game is easy.

Reply:
I am to merge to master right now.  If you want to get ahead of it, it is on develop and you can make the minor changes (I don't think it is anything too fancy so should be easy to do)